### ISSUE TITLE
spectatorhud: Remove weapon icon for dead players

### ISSUE THREAD

**Issue by MVarland** [actionquake/aq2-tng#163]
When a player has died they're shown as holding a MK23 Pistol on the spectatorhud

**Comment by darkshade9** [comment]
Would it make sense to have a skull icon for players who are dead to replace the MK23 Pistol with?

**Comment by MVarland** [comment]
![quake000](https://github.com/actionquake/aq2-tng/assets/104026478/879d2409-c6a4-406f-b2c5-ba4d2359341b)

That might become cluttery since there's already a skull

**Comment by darkshade9** [comment]
Hm true, we could just not render an icon at all?

**Comment by MVarland** [comment]
Sounds good to me

**Comment by darkshade9** [comment]
![image](https://github.com/actionquake/aq2-tng/assets/1392336/3baadac3-2c8a-45c4-a38f-3222a7d719fc)

**Comment by m4son** [comment]
I dont agree with removing a weapon icon., a caster or spectator can see what weapon players are choosing before next round e.g. "owww look at the red team!!! They are going all knives this last round..."

Also from a picture i can tell server is not using latest server release. latest is not using mk23 icon when dead

**Comment by darkshade9** [comment]
The way the code is setup in this branch:
1. If player is dead, don't render an image
2. else, render the curr_weap
3. else, render the chosen weapon

https://github.com/actionquake/aq2-tng/pull/164

I'm open to other suggestions
